helm/README.md

```diff
@@ -0,0 +1,37 @@
+# Kubernetes `pygeoapi` Helm Chart
```

Comment:
@tomkralidis Should we move the NOTES.txt->README.md information into this README.md as it's top level to the Helm deployment? Or prefer keeping them separate?